### PR TITLE
fix: Remove toast messages

### LINKED ISSUE #21490: [Task]: Remove toast messages in Action Execution
### Is there an existing issue for this?

- [X] I have searched the existing issues

### SubTasks

Remove toast messages in Query Execution
Address cypress test cases w.r.t the changes

### PR DESCRIPTION
**Pull request by ChandanBalajiBP**
## Description

> Remove Toast messages on the following scenarios 
>1. Upon execution of an API/Query/JSFunction.
>2. User is in action page and onpageload actions are executed.
>  ex: User is in API page and the API is set to execute on page load. Now if users refresh application then the API is executed. Now we won't show any toasts but open response tab.
> But if user refresh page from canvas or any other page then it's own page , the toasts w.r.t the API execution will be shown.

> TL;DR - Toast messages upon action execution is removed and we are serving this by opening the response tab automatically for the users.

Fixes #21490 

## Type of change

- Bug fix (non-breaking change which fixes an issue)

## How Has This Been Tested?

- Manual
- Cypress

### Dev activity
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] PR is being merged under a feature flag

### QA activity:
- [ ] Test plan has been approved by relevant developers
- [ ] Test plan has been peer reviewed by QA
- [ ] Cypress test cases have been added and approved by either SDET or manual QA
- [ ] Organized project review call with relevant stakeholders after Round 1/2 of QA
- [ ] Added Test Plan Approved label after reveiwing all Cypress test

### CODE REVIEW

**Review by sneha122 — DISMISSED** [review]

File: app/client/cypress/integration/Regression_TestSuite/ClientSideTests/VisualTests/JSEditorIndent_spec.js [hunk 1]
```diff
@@ -224,7 +224,6 @@ myFun2: async () => {
     agHelper.GetNClick("[name='expand-more']", 0, true, 100);
     agHelper.ContainsNClick("myFun2");
     cy.get("div.CodeMirror").matchImageSnapshot("jsObjAfterPrettify2");
-    agHelper.AssertContains("ran successfully");
```
Comment (Aishwarya-U-R) [inline app/client/cypress/integration/Regression_TestSuite/ClientSideTests/VisualTests/JSEditorIndent_spec.js:227]: Instead of removing this - can we add assert to verify its not present?
AssertElementAbsence(locator.specificToast(above message?'))?

Comment (ChandanBalajiBP) [reply app/client/cypress/integration/Regression_TestSuite/ClientSideTests/VisualTests/JSEditorIndent_spec.js:227]: Yep, this makes more sense. I will update these change

**Review by Aishwarya-U-R — COMMENTED** [review]
Cypress changes looks Ok!

**Review by sneha122 — APPROVED** [review]

**Comment by prapullac** [comment]
@ChandanBalajiBP : tested the PR and this looks good to me we can go to the release
